storage/src/checker/mod.rs

```diff
@@ -534,7 +543,7 @@ mod test {
 
     /// Generate a test trie with the following structure:
     ///
-    #[cfg_attr(doc, aquamarine::aquamarine)]
+    #[cfg_attr(doc, aquamarine)]
```

Comment:
Why did this change? Seems to cause a checker dependency warning, and I'm guessing the diagram is not rendered correctly.